libp2p.nimble

```diff
@@ -12,7 +12,7 @@ requires "nim >= 1.6.0",
          "dnsclient >= 0.3.0 & < 0.4.0",
          "bearssl >= 0.1.4",
          "chronicles >= 0.10.2",
-         "chronos >= 3.0.6",
+         "chronos >= 3.0.6 & < 4.0.0",
```

Comment:
Please, let's avoid adding unrelated changes to the PRs.

Reply:
I'll move it.